pairings/Cargo.toml

```diff
@@ -0,0 +1,51 @@
+[package]
+name = "pairings"
```

Comment:
This crate looks like a fork of https://github.com/algorand/pairing-plus via https://github.com/mikelodder7/pairing-plus/
Should we use it instead of the `pairing-plus` crate, in `src/bbs.rs`?
Are the changes planned to be upstreamed?

Reply:
They can’t be right now since pairing-plus is not longer supported. I had submitted a PR to Ursa to reconcile this and it was rejected. The path forward is to leave BBS alone as it’s the 2020 version and the next iteration 2022 won’t use the pairing crate at all once it’s standardized